tests/integration-tests/tests/api/test_api_client.py

```diff
@@ -29,16 +31,32 @@ def _cloudformation_wait(region, stack_name, status):
     waiter.wait(StackName=stack_name)
 
 
-@pytest.mark.usefixtures("region", "os", "instance")
-def test_cluster_operations(request, scheduler, region, pcluster_config_reader, clusters_factory, api_client):
-    cluster_config_path = pcluster_config_reader(scaledown_idletime=3)
+@pytest.mark.usefixtures("os", "instance")
+@pytest.mark.schedulers(["slurm"])
+def test_cluster_slurm(request, region, pcluster_config_reader, api_client):
```

Comment:
can you also inject the scheduler fixture and assert that scheduler==slurm? this because we rely on those fixture to publish test metrics and we want to know this test is related to slurm schedueler

Reply:
I have talked about this with @enrico-usai , are you sure that the fixture is not injected when using this annotation? The reasoning here was that we wanted to "force" the usage of `slurm` here since this is a Slurm specific test case, and we did not want a situation like someone specifying a different scheduler in the test config.

Reply:
Synced up with @demartinofra off-line, reporting here the notes. Configs have precedence over this annotation. I will do as suggested (i.e. take the config value for the schedulers and add an assertion in the test).